Website/src/rendering/src/components/Checkout/LineItemList.tsx

```diff
@@ -0,0 +1,24 @@
+import useOcCurrentOrder from '../../hooks/useOcCurrentOrder';
+import LineItemCard from './LineItemCard';
+
+type LineItemListProps = {
+  editable?: boolean;
+};
+
+const LineItemList = (props: LineItemListProps): JSX.Element => {
+  const { lineItems } = useOcCurrentOrder();
+
+  return lineItems && lineItems.length ? (
+    <ol>
+      {lineItems.map((li) => (
```

Comment:
We try to avoid short variable names that are not indicating their role like `s` and `li` in our projects. Could this one please be changed to `lineItem`?